tests/ocf-zip-comp/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>OCF ZIP container uses compression techniques other than Deflate</title>
+  </head>
+  <body>
+    <p>Test passes if reading system raises error when loading this epub file and this page of the book can not be accessed.</p>
```

Comment:
(I am being overly picky here...).

Very strictly speaking, we do not define what the RS' error behaviour is. It is not conceivable that a RS, whilst reporting an error, makes an attempt to display the content nevertheless. This should be acceptable (while making it clear that this _is_ an erroneous EPUB).

```suggestion
    <p>Test passes if reading system raises error when loading this epub file.</p>
```

Note that the same change should be made on the `ocf-zip-mult` test, too. My apologies not to have realized that before.

Reply:
no problem. Thanks, I will make changes to both.